src/models/helpers/tests/purifyFields.test.js

```diff
@@ -30,6 +30,29 @@ describe('purifyFields', () => {
     expect(instance.set).not.toHaveBeenCalled();
   });
 
+  test('returns if changed does not return an array', () => {
+    const instance = {
+      set: jest.fn(),
+      changed: jest.fn().mockReturnValue('not an array'),
+    };
+
+    purifyFields(instance, ['field1']);
+
+    expect(instance.set).not.toHaveBeenCalled();
+  });
+
+  it('returns if the field value is not a string', () => {
+    const instance = {
+      set: jest.fn(),
+      changed: jest.fn().mockReturnValue(['field1']),
+      field1: 123,
+    };
+
+    purifyFields(instance, ['field1']);
+
+    expect(instance.set).not.toHaveBeenCalled();
+  });
```

Comment:
is there a reason that one is "test" and the other is "it"

Reply:
no good reason